.github/CODEOWNERS

```diff
@@ -1791,7 +1791,9 @@ spec/support/vcr_cassettes/bid @department-of-veterans-affairs/Disability-Experi
 spec/support/vcr_cassettes/brd @department-of-veterans-affairs/lighthouse-dash
 spec/support/vcr_cassettes/carma @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/support/vcr_cassettes/caseflow @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-spec/support/vcr_cassettes/charon @department-of-veterans-affairs/lighthouse-banana-peels
+spec/support/vcr_cassettes/charon/400.yml @department-of-veterans-affairs/lighthouse-banana-peels @department-of-veterans-affairs/backend-review-group
```

Comment:
Charon specific cassettes should either be removed or marked under lighthouse-pivot as it is auth specific.